src/pages/Scholarships.jsx

```diff
@@ -46,33 +48,38 @@ function ScholarshipsPage() {
     });
   };
 
-  const clearQueryParam = (index) => {
-    const newParams = { ...params };
-    delete newParams[index];
-    history.replace({ search: queryString.stringify(newParams) });
+  const pruneQueryParam = (index) => {
+    delete params[index];
+    history.replace({ search: queryString.stringify(params) });
   };
 
+  if (
+    minAmount !== undefined &&
+    !(Number.isInteger(minAmount) && minAmount > 0)
+  ) {
+    pruneQueryParam(QUERY_PARAM_MIN_AMOUNT);
+  }
+
+  if (
+    maxAmount !== undefined &&
+    !(Number.isInteger(maxAmount) && maxAmount > 0)
+  ) {
+    pruneQueryParam(QUERY_PARAM_MAX_AMOUNT);
+  }
+
   useEffect(() => {
+    if (maxAmount > 0 && maxAmount <= minAmount)
+      setError('The minimum amount must be less than the Maximum.');
```

Comment:
Don't load the scholarships in this case. Also, I think it's ok if the min/max are the same.
```suggestion
    if (maxAmount && maxAmount < minAmount) {
      setError('The minimum amount must be less than the Maximum.');
      return;
    }
```